.github/workflows/rag-production-sample.yml

```
@@ -0,0 +1,274 @@
+            echo "| Avg context recall | \`${AVG_CONTEXT_RECALL:-—}\` |"
+            echo "| Avg context precision | \`${AVG_CONTEXT_PRECISION:-—}\` |"
+          } >> "$GITHUB_STEP_SUMMARY"
+
+      # ---------------------------------------------------------------------
+      # Step 4 — Slack notification (success path). Failure path follows.
+      # ---------------------------------------------------------------------
+      - name: Notify #app-development-analytics — success
+        if: ${{ success() }}
+        uses: ./.github/actions/send-slack-notification
+        with:
+          channel: APP_DEV_ANALYTICS
```

actionlint (1.7.12)

[error] input "channel" is not defined in action "Send Slack Notification" defined at "./.github/actions/send-slack-notification". available inputs are "actor", "dedup_key", "dedup_window_minutes", "dry_run", "fields", "max_retries", "message", "run_url", "status", "verbiage", "webhook_url", "workflow_name"

(action)